EpamWebSiteTestingFramework/Epam.TestAutomation.Pages/PageObjects/Pages/JobSearchResultPage.cs

```diff
@@ -0,0 +1,42 @@
+using Epam.TestAutomation.Core.BasePage;
+using Epam.TestAutomation.Core.DriverCreator;
+using Epam.TestAutomation.Core.Elements;
+using Epam.TestAutomation.Core.Helper;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Pages.PageObjects.Pages;
+
+public class JobSearchResultPage : BasePage
+{
+    public ElementsList FoundResultList => new(By.XPath("//*[@class='search-result__item-name']"));
+
+    public Label SearchResultHeadingTitle => new Label(By.XPath("//*[@class='search-result__heading']"));
+
+    public Label ErrorMessageWhenNoJobsFound =>
+        new Label(By.XPath("//*[@class='search-result__error-message' and @role ='alert']"));
```

Comment:
please use contains when you create locator for class as those locators are unstable

Reply:
//*[@class='search-result__error-message' and contains(@role, 'alert')]